manager/integration/tests/test_rwx.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+'''
+Fixture required for this module to clean up longhorn-nfs-provisioner related
+resources.
+'''
+
+
+@pytest.mark.skip(reason="TODO")
+def test_rwx_with_statefulset_multi_pods():
+    """
+    Test writing of data in same volume from 2 pods
+
+    1. Create PodSecurityPolicy for longhorn-nfs-provisioner as sample yaml
+       https://raw.githubusercontent.com/longhorn/longhorn/master/examples
+       /rwx/01-security.yaml
+
+    2. Create serviceAccount, service, longhorn-nfs-provisioner deployment,
+       PVC and StorageClass as sample yaml https://raw.githubusercontent.com/
+       longhorn/longhorn/master/examples/rwx/02-longhorn-nfs-provisioner.yaml
+
+    3. Wait for longhorn-nfs-provisioner deployment to come up healthy.
+    4. Verify the volume named as the PV attached to longhorn-nfs-provisioner
+       `volume.ready == True`
+    5. Create a StatefulSet of 2 pods with VolumeClaimTemplate and
+       longhorn-nfs class in default namespace.
+    6. Wait for both pods to come up running.
+    7. Verify two folders get created with PV names in longhorn-nfs-provisioner
+       volume.
+    8. Write data in both pods and compute md5sum.
```

Comment:
How do we do this?
Maybe map the export folder from the nfs provisioner?

Reply:
Having pods name and their data path, we can write data to pods.